### PR TITLE
Sword fix

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: app-text/sword/sword-1.8.1.recipe [hunk 1]
```diff
@@ -38,16 +38,18 @@ PROVIDES="
...
 	lib:libsword_$portVersion$secondaryArchSuffix
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	icu$secondaryArchSuffix
-#	lib:libcppunit$secondaryArchSuffix
+	lib:libicudata$secondaryArchSuffix
+	lib:libicui18n$secondaryArchSuffix
+	lib:libicuio$secondaryArchSuffix
+	lib:libicuuc$secondaryArchSuffix
```
Comment (korli) [inline app-text/sword/sword-1.8.1.recipe]: libicu* come after libcurl

**Comment by korli** [comment]
still one remark, otherwise OK. Thanks!

File: app-text/sword/sword-1.8.1.recipe (hunk 1 above)
Comment (Begasus) [reply app-text/sword/sword-1.8.1.recipe]: Bugger, should've seen it :)

**Comment by Begasus** [comment]
Think this should be ok :) (not checked on x86_64 but shouldn't be any structural changes now)

**Comment by korli** [comment]
thanks!

**Pull request merged**